eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -26,6 +26,7 @@ stages:
     - template: ../jobs/archetype-sdk-tests.yml
       parameters:
         TestName: Emulator
+        DisableAzureResourceCreation: true
```

Comment:
I suggest using `EnableAzureResourceCreation` and have it default to `true` in the `archetype-sdk-tests.yml` file... though this is purely cosmetic and individual preference. The reason behind this is that it translates to "yes, enable azure resource creation" as opposed to the double negative: "don't disable azure resource creation"